services/core-api/tests/factories.py

```diff
@@ -577,7 +577,7 @@ def bonds(obj, create, extracted, **kwargs):
             return
 
         if not isinstance(extracted, int):
-            extracted = random.randint(0, 3)
+            extracted = random.randint(1, 3)
```

Comment:
I believe we want to create some permits that don't yet have bonds attached to them. It's good to have test data for that case. I did have a discussion with @Jsyro about this when I added this to the factories.

Reply:
I changed this specifically because when I was trying to create a permit to test bonds I would sometimes get a permit created with now bonds. That might not be the best way to do it but I wasn't super sure on how to pass in the argument needed to guarantee a bond is created.